forms-flow-web/src/components/Form/List.js

```diff
@@ -345,6 +355,15 @@ const List = React.memo((props) => {
 
   return (
     <>
+      <ReleaseNoteModal 
+        showModal={showReleaseNoteModal} 
+        onClose={() => {
+          setShowReleaseNoteModal(false);
+          dispatch(readReleaseNote({release_note_id: releaseNoteData.data?.id}));
+          }} 
```

Comment:
Just make sure after it is read (`onClose`), the unreadNote is removed from the state (by resetting or fetching unread again), so if a user returned to this page, the modal does not pop up.

Reply:
Correct Iman, The ReleaseNote object is set to blank when clicking on "onCLose". even if API requests succeed or fail.